src/EnergyPlus/HVACSingleDuctInduc.cc

```diff
@@ -764,7 +795,11 @@ namespace HVACSingleDuctInduc {
 				}
 			} else {
 				CheckZoneSizing( IndUnit( IUNum ).UnitType, IndUnit( IUNum ).Name );
-				MaxTotAirVolFlowDes = max( TermUnitFinalZoneSizing( CurZoneEqNum ).DesCoolVolFlow, TermUnitFinalZoneSizing( CurZoneEqNum ).DesHeatVolFlow );
+				if ( CurTermUnitSizingNum > 0 ) {
```

Comment:
Maybe the checks for CurTermUnitSizingNum don't need to be there, but early on in this development, there were some timing issues of when that got initialized and these were added to protect against trying to access TermUnitFinalZoneSizing(0).